apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/PgNumericTests.cs

```diff
@@ -329,56 +330,104 @@ public void FromDecimal(decimal input, string expectedValue)
             Assert.Equal(expectedValue, numeric.ToString());
         }
 
-        [Theory]
-        [InlineData("0.00")]
-        [InlineData("1.00")]
-        [InlineData("1.01")]
-        [InlineData("12345.123456000")]
-        [InlineData("-0.000001")]
-        [InlineData("1234567891.123456789123")]
-        [InlineData("-12345678912.12345678912")]
-        [InlineData("123456789123456789.12345678912")]
-        [InlineData("12345678912345678912345678.1234567891234567891234")]
-        [InlineData("9999999999999999999999999999.999999999999999999999999")]
-        [InlineData("100")]
-        [InlineData("1.33333333333333333")]
-        public void ToDecimal(string input)
```

Comment:
I've removed this test entirely as this is precisely what we stopped doing, i.e. rounding as decimal does, instead we are now truncating.

Some of the relevant test cases from here I've moved to the other tests.